doc/en/components/data-grid-row-paging.md

```diff
@@ -100,4 +130,12 @@ private onPageChanged = (pageNumber: number, data: any[]) => {
 };
 ```
 
+## React Pagination and Grid Interactions 
+
+When users interact with the Grid like sorting and grouping, and you have enabled the React Pager component on the grid, you need to use the following functions to ensure that the React Pagination data is synchronized with the React table display. 
```

Comment:
use $Platform$ variable instead of "React"

Reply:
done